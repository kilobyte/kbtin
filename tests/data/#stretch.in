#loop 1,6 {#split in {} {a b c d e f} $0;#loop 1,6 {#stre out $0 $in;#show $out}}
