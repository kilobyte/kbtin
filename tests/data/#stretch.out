a
a a
a a a
a a a a
a a a a a
a a a a a a
a
a b
a a b
a a b b
a a a b b
a a a b b b
a
a c
a b c
a a b c
a a b b c
a a b b c c
a
a c
a c d
a b c d
a a b c d
a a b c c d
a
a d
a c e
a c d e
a b c d e
a a b c d e
a
a d
a c e
a c d f
a c d e f
a b c d e f
